ios/RNGoogleCast/RNGoogleCast.m

```diff
@@ -241,6 +268,15 @@ - (void)stopObserving {
   }
 }
 
+#pragma mark - GCKRequestDelegate
+
+- (void)requestDidComplete:(GCKRequest *)request {
+    if (request == styleChangeRequest) {
+        NSLog(@"Style update completed.");
+        styleChangeRequest = nil;
```

Comment:
Hmm, any reason to keep the request around other than logging?

Reply:
Not really, this is unneeded!